src/cleed++/Core/Coordinate.cc

```
@@ -0,0 +1,127 @@
+  ::coord_set_x(this->pos, x);
+  ::coord_set_y(this->pos, y);
+  ::coord_set_z(this->pos, z);
+}
+
+Coordinate::Coordinate(const Coordinate &position)
+{
+  this->pos = ::coord_init();
+  ::coord_copy(this->pos, position.pos);
+}
+
+Coordinate::Coordinate(const coord_t *position)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: pos

(cppcoreguidelines-pro-type-member-init)